src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/model/translator.kt

```diff
@@ -76,8 +77,9 @@ fun Prisoner.translate(ob: OffenderBooking, incentiveLevel: IncentiveLevel?) {
     ob.sentenceDetail?.conditionalReleaseOverrideDate ?: ob.sentenceDetail?.conditionalReleaseDate
   this.actualParoleDate = ob.sentenceDetail?.actualParoleDate
 
-  this.locationDescription = ob.locationDescription
-
+  this.locationDescription = restrictivePatient
+    ?.let { "${ob.locationDescription} - discharged to ${it.dischargedHospital?.description}" }
+    ?: ob.locationDescription
   // get the most serious offence for this booking
```

Comment:
equivalent to this code `return booking.copy(
       locationDescription = booking.locationDescription + " - discharged to " + restrictivePatient.hospitalLocation.description,
       restrictivePatient = RestrictivePatient(
  `

which is now moved to correct place